spec.html

```diff
@@ -10015,6 +10015,16 @@ <h1>Static Semantics: AssignmentTargetType ( ): ~simple~ or ~invalid~</h1>
         1. Let _expr_ be the |ParenthesizedExpression| that is covered by |CoverParenthesizedExpressionAndArrowParameterList|.
         1. Return the AssignmentTargetType of _expr_.
       </emu-alg>
+      <emu-grammar>
+        CallExpression :
+          CoverCallExpressionAndAsyncArrowHead
+          CallExpression Arguments
+      </emu-grammar>
+      <emu-alg>
+        1. [id="step-assignmenttargettype-web-compat-insertion-point", normative-optional] If the host is a web browser or otherwise supports <emu-xref href="#sec-late-errors-for-function-call-assignment-targets" title></emu-xref>, then
+          1. If IsStrict(this |CallExpression|) is *false*, return ~web-compat~.
```

Comment:
This guard can still be combined with the containing guard.

Reply:
Ah sure. Based on #2952, I thought we wanted it this way, but that works.

Reply:
Eh, we're just following that high-level strategy. #2952 hasn't been reviewed for small things like this yet.

Reply:
Cool. Lemme know if this is okay wrt "or" and "and" usage. :bow:

Reply:
LGTM. We'll need to see what the other editors think.